### DISCUSSION TITLE
face:new search shows faces I have already tagged.

### DISCUSSION THREAD

**Discussion by Richie-BTC-Rich** [photoprism/photoprism#2030] [General]
If I go to People the recognized tag shows all the people my photos. Then if I go to the new tab it says "no people found" with a button to "SHOW ALL NEW FACES" Clicking on this button shows faces that I have all ready labeled.  

Am I misunderstanding how this works ?

**Comment by lastzero** [comment]
Are you sure there are no additional faces on theses images that have not been assigned to a person yet?

**Reply by Richie-BTC-Rich** [reply]
Yes in one particular photo its almost entirely a face, Looking the photo details > People tab shows it as labeled as that person

**Reply by lastzero** [reply]
Did you read this already? https://docs.photoprism.app/user-guide/organize/people/

**Reply by Richie-BTC-Rich** [reply]
Yes several times.  I must be missing something I expected the searching face:new would show me photos of people that were "untagged"

**Reply by Richie-BTC-Rich** [reply]
I just discovered im running a pretty old version 210925-96168e4b.  Maybe I should figure out how to upgrade and check it out then.

**Reply by marcelbrueckner** [reply]
I'm running a fresh install of a recent version (16th Nov. 2022) and have the same issue. By now it's just a single image, every other tagged images disappear from the `faces:new` page after being tagged. Untagging and re-tagging the image didn't help.

**Reply by lastzero** [reply]
Maybe there's a second face in the image which has not been assigned yet?

**Reply by marcelbrueckner** [reply]
Actually, there is another half of a face in the picture. But this wasn't detected as such, so I can't assign a person. Only one face has been detected by PhotoPrism (and this has been assigned).

**Reply by lastzero** [reply]
OK, then that's strange. Maybe it was hidden? Can't tell you without seeing it... If you know how it can be reproduced, you may open a bug report with all the details 👍

**Reply by marcelbrueckner** [reply]
I "accidentally" discovered that my image was part of a stack. The stack's other image had a person detected (but not recognized/assigned). After unstacking, `face:new` works as expected. The previous stack's first image (on which a person was assigned) doesn't show up anymore. Only the previous stack's other image remain. After assigning a person it doesn't show up anymore as well.

The person on my image is a child if that matters. I've read that detecting/recognizing kids is somewhat unreliable/hard due to the fact that training data usually doesn't contain children.

**Reply by comminutus** [reply]
This sounds similar to what I've been experiencing as well. I have a bug report here: #2936
